Documentation/community/roadmap.rst

```diff
@@ -14,35 +14,60 @@ broader community. You'll also find here some pointers on how you can
-++----------------------------+------------------------------------------------------------+
-|| Network Policy             | Stable                                                     |
-++----------------------------+------------------------------------------------------------+
-|| Kube-proxy Replacement     | Stable                                                     |
-++----------------------------+------------------------------------------------------------+
-|| Egress Gateway             | Stable                                                     |
-++----------------------------+------------------------------------------------------------+
-| Multi-Cluster (ClusterMesh) | Stable (:ref:`Roadmap Details<rm-clustermesh>`)            |
-+-----------------------------+------------------------------------------------------------+
-| Hubble Observability        | Stable (:ref:`Roadmap Details<rm-hubble-observability>`)   |
-+-----------------------------+------------------------------------------------------------+
-| Service Mesh                | Stable (:ref:`Roadmap Details<rm-cilium-service-mesh>`)    |
-+-----------------------------+------------------------------------------------------------+
-| Tetragon Security           | Beta                                                       |
-+-----------------------------+------------------------------------------------------------+
+
++--------------------------------------------------+----------------------------------------------------------+
+| eBPF Networking                                  | Stable (:ref:`Roadmap Details<rm-advanced-networking>`)  |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`Kubernetes CNI<k8s_network_root>`         | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| `Load Balancing`_                               | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`k8s_policy`                               | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`Kube-proxy Replacement<kubeproxy-free>`   | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`egress-gateway`                           | Stable                                                   |
+++-------------------------------------------------+----------------------------------------------------------+
+|| :ref:`gsg_encryption` (IPSec and WireGuard)     | Stable                                                   |
```

Comment:
There's some caveats here I think, for instance for node-to-node encryption? I'm not sure whether the level of testing really validates a stable tag yet or not cc @cilium/sig-encryption

Reply:
I added a bullet in the roadmap details below to cover graduating node-to-node WireGuard to stable. I considered adding something in the table referring to the docs for details, but it seemed like that might well also apply to other features, and it would end up being too much detail for this high-level view